src/Database/Postgres/Temp/Internal/Core.hs

```diff
@@ -128,6 +128,7 @@ teeHandle orig f =
     outputRef <- newIORef []
 
     let readerLoop = forever $ do
+          threadDelay 1000000
```

Comment:
👍

Reply:
Yeah; just added this temporarily to show where it's racy